kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_slip.h

```diff
@@ -146,10 +146,11 @@ class ResidualBasedEliminationBuilderAndSolverSlip
         // Validate default parameters
         Parameters default_parameters = Parameters(R"(
         {
-            "domain_size" : 3,
-            "variable_x" : "VELOCITY_X",
-            "variable_y" : "VELOCITY_Y",
-            "variable_z" : "VELOCITY_Z"
+            "builder_and_solver_type" : "ResidualBasedEliminationBuilderAndSolverSlip",
```

Comment:
why don't you use `name` or `type` for this?

Should simplify the handling in the factories and also avoid name-clases

Reply:
Well, because when constructing a whole strategy you need to differentiate them

Reply:
In favor of @philbucher comment

```json
builder_and_solver:{
    "name": "...."
}

Reply:
Ok, but this will require some extra work